apps/website/common/settings.ts

```diff
@@ -0,0 +1,66 @@
+import { MenuItem, SocialLink } from "@/types"
+import { Icons } from "@/components/elements/Icons"
+
+export const APP_SETTINGS = {
+    APP_TITLE: "Bandada",
+    APP_DESCRIPTION:
+        "plug-and-play infrastructure that empowers anyone to create and manage privacy-preserving groups of anonymous individuals."
+}
+
+export const MENU_ITEMS: MenuItem[] = [
+    {
+        label: "Projects",
+        href: "/projects"
+    },
+    {
+        label: "Build",
+        href: "/build-with-bandada"
```

Comment:
```suggestion
        href: "/build"
```

I think it's ok to name it `build`, we just have to add `!app/build` to the website `.gitignore` file. The folder name inside `app` should be updated too.